hummingbird.yml

```diff
@@ -13,4 +13,5 @@
 - hosts: andrewd
   become: yes
   roles:
+    - { role: common, binary_only: yes, rage: ['common'] }
```

Comment:
RAGE!

Reply:
I think that is the funniest typo ever :)